web/client/themes/default/icons.less

```diff
@@ -1,694 +1,694 @@
-@charset "UTF-8";
```

Comment:
Hello, why this file has changed?

Reply:
@vlt1 please align to latest mapstore and sync with @allyoucanmap , we changed the icons generation procedure recently.